tools/modules/EnsEMBL/Web/ConfigPacker.pm

```diff
@@ -154,4 +156,22 @@ sub _vep_config_warning {
   warn $message." - thrown by ".__FILE__."\n";
 }
 
+sub _configure_assembly_converter_files {
+  my $self = shift;
+  my $species = $self->species;
+  my $version = $SiteDefs::ENSEMBL_VERSION;
```

Comment:
doesn't look like we are using `$version` elsewhere? do we need it? I would have thought we need the release version for the path but not sure if `ENSEMBL_CHAIN_FILE_DIR` already has it

Reply:
Yes it does: 

$SiteDefs::ENSEMBL_CHAIN_FILE_DIR           = defer { sprintf '/net/isilonP/public/ro/ensweb-data/%stools/%s/%s/assembly_converter', $SiteDefs::USE_SNAPSHOT_ENSWEBDATA ? 'latest/' : '', $SiteDefs::SUBDOMAIN_DIR, $release };

I think we can ommit line 162.